xds/src/test/java/io/grpc/xds/XdsClientWrapperForServerSdsTestMisc.java

```diff
@@ -202,6 +202,17 @@ public void startXdsClient_expectException() {
     verify(mockServerWatcher, never()).onError(any(Throwable.class), eq(false));
   }
 
+  @Test
+  public void xdsClientStart_multipleReplacements() {
```

Comment:
I  don't see any value of such a test, it's majorly testing the functionality of `String.replaceAll(...)`...

Reply:
The test verifies the contract and is useful and is not there to test functionality of replaceAll. I want the test to show breakage when implementation changes.